gaptrain/md.py

```diff
@@ -36,6 +36,62 @@ def simulation_steps(dt, kwargs):
     return max(int(time_fs / dt), 1)
 
 
+def ase_momenta_string(configuration, temp, bbond_energy, fbond_energy):
```

Comment:
A doc string would be useful here are you have different object types

Reply:
💯